src/constants/languages.ts

```diff
@@ -0,0 +1,8 @@
+interface LanguageMap {
+  [key: string]: string;
+}
+
+export const languages: LanguageMap = {
```

Comment:
Working with languages this way is because of Legacy OSE decisions which shouldn't be part of OCX 1.0, where each language should have its own separate OCX. So, let's keep this inside legacy OSE specific code.